src/app/pages/common/entity/table/expandable-table/expandable-table.component.ts

```diff
@@ -50,11 +68,37 @@ export class ExpandableTableComponent {
     };
   }
 
+  ngAfterViewChecked(): void {
+    if (this._tableConf.limitRows
+      && this.appTable.nativeElement.children[0].children[1].children[0].children[0].children[0].children[0].children[0].children[0]) {
+      const tableHeaderHeight = this.appTable.nativeElement.children[0].children[1].children[0].children[0].children[0].children[0].children[0].children[0].children[0].offsetHeight;
```

Comment:
![source-2](https://user-images.githubusercontent.com/1481270/119543322-357d9280-bd99-11eb-97ff-91f28ec4ceb8.gif)

Reply:
Okay,
1. This is hard to read.
2. This will break if DOM structure changes.

Can you at least do something like querySelector()? 
I will also be okay with this being a separate directive.